src/sass/buttons.scss

```diff
@@ -0,0 +1,113 @@
+
+.app__create-button,
+.add-button,
+.yes-button,
+.cancel-button {
+  display: inline-block;
+  margin-top: var(--flow-space, 1.7rem);
+  padding: 0.3rem 2.5rem;
+  border: 2px solid;
+  border-radius: 0.375rem;
+  box-shadow: 6px 6px 0 $color-secondary;
+  background:$bg-general ;
+  color: $color-primary;
+  font-family: $font-codeSaver;
+  font-size: 0.8rem;
+  font-weight: 900;
+  line-height: 1.5;
+  letter-spacing: 0.05ch;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.app__create-button {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 0.3125rem 0.3125rem 0.3125rem 0.9rem;
+}
+
+@media (min-width: 48em) {
```

Comment:
For media queries, you could use a mixin. For example:  

``
@mixin break($length) {
    @media (min-width: $length) {
      @content;
    }
 }``

And then in the code, wherever breakpoints are used, you could do smth like this:

``
.app__create-button {
      display: flex;
      align-items: center;
      width: 100%;
      padding: 0.3125rem 0.3125rem 0.3125rem 0.9rem;
     @include break(48em) {
        width: 15rem;
        padding: 0.8rem 2.5rem;
        letter-spacing: 0.05ch;
     }
     @include break(64em) {
        width: 17rem;
        font-size: 1rem;
     }
    }``

Reply:
Thanks @katia-utochkina, I have added the mixin.